samples/adc_mux/main.cpp

```diff
@@ -0,0 +1,31 @@
+#include <EVT/io/GPIO.hpp>
+#include <EVT/io/manager.hpp>
+#include <EVT/dev/ADCMux.hpp>
+#include <EVT/utils/time.hpp>
+
+namespace IO = EVT::core::IO;
+namespace DEV = EVT::core::DEV;
+namespace time = EVT::core::time;
+
+int main() {
+    IO::init();
+
+    IO::GPIO& ctrlGPIO1 = IO::getGPIO<IO::Pin::PB_10>(IO::GPIO::Direction::OUTPUT);
+    IO::GPIO& ctrlGPIO2 = IO::getGPIO<IO::Pin::PB_4>(IO::GPIO::Direction::OUTPUT);
+
+    IO::ADC& adc = IO::getADC<IO::Pin::PB_8>();
+
+    DEV::ADCMux mux(ctrlGPIO1, ctrlGPIO2, adc);
+
+    IO::GPIO& ctrlGPIO3 = IO::getGPIO<IO::Pin::PB_8>(IO::GPIO::Direction::OUTPUT);
```

Comment:
What are you using `ctrlGPIO3` for? And why is it the same pin as the `adc`?

Reply:
Sorry, ctrlGPIO3 we created just to test things we will be resolving this.